test/python/algorithms/minimum_eigensolvers/test_adapt_vqe.py

```diff
@@ -0,0 +1,141 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test of the AdaptVQE minimum eigensolver """
+
+import unittest
+from test.python.algorithms import QiskitAlgorithmsTestCase
+
+from qiskit.algorithms.minimum_eigensolvers import VQE
+from qiskit.algorithms.minimum_eigensolvers.adapt_vqe import AdaptVQE, TerminationCriterion
+from qiskit.algorithms.optimizers import SLSQP
+from qiskit.circuit import QuantumCircuit, QuantumRegister
+from qiskit.circuit.library import EvolvedOperatorAnsatz
+from qiskit.opflow import PauliSumOp
+from qiskit.opflow.operator_globals import X, Y
```

Comment:
We have been trying to avoid opflow in the new algorithms and in tests (aside from PauliSumOp)